projects/plugins/boost/app/lib/Features_Status.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Class to check the status of Boost features.
+ *
+ * @package automattic/jetpack-boost
+ */
+
+namespace Automattic\Jetpack_Boost\Lib;
+
+use Automattic\Jetpack_Boost\Lib\Critical_CSS\Critical_CSS_State;
+
+/**
+ * Class that handles checking the status of Jetpack Boost features.
+ */
+class Features_Status {
+
+	const GOOD = 'good';
+	const BAD  = 'bad';
+
+	public static function get_total_problems() {
+		$status = self::get_cloud_css_state_status();
+		return $status === self::GOOD ? 0 : 1;
+	}
+
+	private static function get_cloud_css_state_status() {
+		$is_enabled = '1' === get_option( 'jetpack_boost_status_critical-css' );
```

Comment:
Though this is precisely what we need, there is an abstraction we use for module statuses, and I think we should continue to use them rather than checking the option directly.
This would be the recommended way:

```
use Automattic\Jetpack_Boost\Features\Optimizations\Critical_CSS\Critical_CSS;
use Automattic\Jetpack_Boost\Lib\Status;

$status = new Status( Critical_CSS::get_slug() );
if( $status->is_enabled() ) { ... }
```